Bottom: 24ee05b22633ceae43af4e463136069a47fafeab
Top:    513f13f516c0c3f12974e1594274de58f77a5906
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 00:05:11 +0100

constify name argument to ghid_notebook_page()

Avoids a compiler warning when building the GTK hid.


---

diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index 4e1bf60..887128f 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -617,7 +617,7 @@ ghid_scrolled_selection (GtkTreeView * treeview, GtkWidget * box,
 }
 
 GtkWidget *
-ghid_notebook_page (GtkWidget * tabs, char *name, gint pad, gint border)
+ghid_notebook_page (GtkWidget * tabs, const char *name, gint pad, gint border)
 {
   GtkWidget *label;
   GtkWidget *vbox;
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 2bbd2de..6976074 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -412,7 +412,7 @@ GtkWidget *ghid_framed_vbox_end (GtkWidget * box, gchar * label,
 GtkWidget *ghid_category_vbox (GtkWidget * box, const gchar * category_header,
 			       gint header_pad, gint box_pad,
 			       gboolean pack_start, gboolean bottom_pad);
-GtkWidget *ghid_notebook_page (GtkWidget * tabs, char *name, gint pad,
+GtkWidget *ghid_notebook_page (GtkWidget * tabs, const char *name, gint pad,
 			       gint border);
 GtkWidget *ghid_framed_notebook_page (GtkWidget * tabs, char *name,
 				      gint border, gint frame_border,
